### ISSUE TITLE
SUGGESTION FOR THE POOTLE TRANSLATE SERVICE

### ISSUE THREAD

**Issue by OfficalAhmad** [translate/pootle#6805]
So i was thinking of how there has been many cases of users attempting to register for a new Pootle account by requesting for one on the localisation mailing list (L10N@apache.openoffice.org). The problem persists where users are given the reset password link, and then waiting for the email to reply in order to be able to reset their password. I was thinking now it is probably annoying those users and the admins, and perhaps, to make the whole registration process so much easier.... why not give users the ability to create a Pootle account themselves, and this will help them set a password from the beginning, without the admins having to always look at the same persisting problem with why the reset password email reply not functioning.

So, to sum up, my suggestion is create a registration form for the Pootle website, and enable users to create their accounts themselves, in order for them to have a set password from the beginning. This also lessens the localisation mailing list being filled with emails about requests for new Pootle accounts (not that it is annoying, but to avoid the reset password link issue).
